tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/AspNet/AspNetAsyncHandlerTests.cs

```diff
@@ -0,0 +1,70 @@
+// <copyright file="AspNetAsyncHandlerTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+using Datadog.Trace.TestHelpers;
+using FluentAssertions;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests.AspNet
+{
+#if NET461
```

Comment:
Should/can we test this on `net451` too?

Reply:
Yes, we should test this on net45 as well since the AsyncLocal storage mechanism is different between the two builds. I'll follow up with that on another PR